[Fluent] Update Avalonia Behaviors package and use new responsive behavior

wieslawsoltes opened the pull request:
- [x] Update Avalonia.Xaml.Behaviors to Version 0.10.6.7
- [x] Use new AdaptiveBehavior/AdaptiveClassSetter for responsive design (instead of DataTriggerBehavior/AddClassAction/RemoveClassAction)
- [x] Remove AddClassAction and RemoveClassAction - have been added into Avalonia.Xaml.Interactions package

Example usages of new responsive behavior:
https://github.com/wieslawsoltes/AvaloniaBehaviors/blob/master/samples/BehaviorsTestApplication/Views/Pages/AdaptiveBehaviorView.axaml

soosr commented:
```
<ir:ResponsiveClassSetter Minimum="0" Maximum="550" ClassName="narrow"/>
```
The only way to invert the logic is this?
```
<ir:ResponsiveClassSetter Minimum="550" Maximum="Infinity" ClassName="narrow"/>
```

wieslawsoltes commented:
> ```
> <ir:ResponsiveClassSetter Minimum="0" Maximum="550" ClassName="narrow"/>
> ```
> 
> The only way to invert the logic is this?
> 
> ```
> <ir:ResponsiveClassSetter Minimum="550" Maximum="Infinity" ClassName="narrow"/>
> ```

You can set e.g.: `MinimumOperator="GreaterThanOrEqual"` `MaximumOperator="LessThan"` or any other available operator:
https://github.com/wieslawsoltes/AvaloniaBehaviors/blob/fbf964878d6790ba58214003d9c32ff53a09635d/src/Avalonia.Xaml.Interactions/Core/ComparisonConditionType.cs#L6-L32

wieslawsoltes commented:
> ```
> <ir:ResponsiveClassSetter Minimum="0" Maximum="550" ClassName="narrow"/>
> ```
> 
> The only way to invert the logic is this?
> 
> ```
> <ir:ResponsiveClassSetter Minimum="550" Maximum="Infinity" ClassName="narrow"/>
> ```

@soosr You can also use `BoundsProperty="Height"` to respond to height changes.

wieslawsoltes commented:
One thing to add into `ResponsiveControlBehavior` is support for observing/setting different control and setting style for different control too.

```XAML
<ir:ResponsiveClassSetter Minimum="0" Maximum="80" SourceControl="{Binding #StackPanel}" TargetControl="{Binding {Binding #TextBlock}" ClassName="narrow"/>
```

if `SourceControl` or/and `TargetControl` are null then `AssociatedObject` is used.

wieslawsoltes commented:
@soosr Can you give another review, I have renamed and refactored behaviors to enable more complex scenarios for adaptive and responsive layouts, examples:
https://github.com/wieslawsoltes/AvaloniaBehaviors/blob/1a4d541215766a2e353e159c50559dac8eba19af/samples/BehaviorsTestApplication/Views/Pages/AdaptiveBehaviorView.axaml#L33-L39
https://github.com/wieslawsoltes/AvaloniaBehaviors/blob/1a4d541215766a2e353e159c50559dac8eba19af/samples/BehaviorsTestApplication/Views/Pages/AdaptiveBehaviorView.axaml#L94-L103

You can now set condition for MinWidth/MaxWidth and/or MinHeigth/MaxHeight is adaptive setter and set independently SourceControl/TargetControl in adaptive behavior (you can also override TargetControl for each adpative setter).

soosr reviewed (approved):
tACK
